fakeprod/datarepo-api.yaml

```diff
@@ -9,6 +9,13 @@ env:
   DB_STAIRWAY_USERNAME: drmanager
   DB_STAIRWAY_URI: jdbc:postgresql://terra-jade-gcloud-sqlproxy.data-repo:5432/stairway
   DB_DATAREPO_URI: jdbc:postgresql://terra-jade-gcloud-sqlproxy.data-repo:5432/datarepo
+  DATAREPO_RESOURCEID: datarepo-terra
+  DATAREPO_DNSNAME: jade-terra.datarepo-prod.broadinstitute.org
+  DATAREPO_GCS_BUCKET: broad-datarepo-terra-prod
+  DB_STAIRWAY_FORCECLEAN: false
+  SAM_BASEPATH: https://sam.dsde-prod.broadinstitute.org
+  SAM_STEWARDSGROUPEMAIL: Stewards@firecloud.org
```

Comment:
De we have/steward's group specified for the dev environment?

Reply:
I think that is a great question, but as I looked into it, I think it opens a can of worms. 
This group is specified for the dev environment in application.properties, along with a lot of other environment specific variables. I think this is definitely something we should tackle, but I think I've already let the scope creep on this ticket too much.  I added another ticket to cover [this](https://broadworkbench.atlassian.net/browse/DR-1301).  Let me know what you think of this plan.

Reply:
Sounds good to me